include/libopencm3/stm32/l0/timer.h

```diff
@@ -138,31 +138,53 @@ LGPL License Terms @ref lgpl_license
 #define LPTIM_IER_UPIE			(1 << 5)
 #define LPTIM_IER_DOWNIE		(1 << 6)
 
+/* --- LPTIM_CFGR values ----------------------------------------------------- */
+
+/* CKSEL: Select internal (0) or external clock source (1) */
```

Comment:
you need a /** to make doxygen pick it up

Reply:
ahhh, indeed. thanks